serialio/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "2.4.0"
+__version__ = "2.4.1"
```

Comment:
Please just fix the code.
I'll bump the version myself on another PR (probably with other bug fixes)

Reply:
Done